### DISCUSSION TITLE
Azure Pipelines are green

### DISCUSSION THREAD

**Discussion by cenit** [microsoft/vcpkg#22109] [General]
How much happy are you @BillyONeal (I suppose you are the most involved in CI)? Personally, I am a lot, it took a very long time to reach this status and I hope that from now on it will be the norm and not an exception.
Congratulations to the whole vcpkg team 🥳 🎉

**Comment by BillyONeal** [comment]
Mostly this has been all contributors flushing out optional dependencies in ports to make their builds more repeatable. This isn't just a win for our CI, it's also an indication that the ports tree is more reliable for customers.

Thanks to everyone who has submitted changes to improve resiliency of ports!
